concent_api/api-e2e-force-accept-or-reject-results-test.py

```diff
@@ -11,30 +10,27 @@
 
 from common.helpers import get_current_utc_timestamp
 from common.helpers import sign_message
-from common.testing_helpers import generate_priv_and_pub_eth_account_key
-from core.utils import calculate_maximum_download_time
 
 from api_testing_common import api_request
+from api_testing_common import get_cluster_name_from_address
 from api_testing_common import count_fails
 from api_testing_common import create_client_auth_message
 from api_testing_common import create_signed_report_computed_task
 from api_testing_common import create_signed_subtask_results_accepted
 from api_testing_common import create_signed_task_to_compute
-from api_testing_common import PROVIDER_PRIVATE_KEY
-from api_testing_common import PROVIDER_PUBLIC_KEY
-from api_testing_common import REPORT_COMPUTED_TASK_SIZE
-from api_testing_common import REQUESTOR_ETHEREUM_PRIVATE_KEY_FOR_EMPTY_ACCOUNT
-from api_testing_common import REQUESTOR_ETHEREUM_PUBLIC_KEY_FOR_EMPTY_ACCOUNT
-from api_testing_common import REQUESTOR_PRIVATE_KEY
-from api_testing_common import REQUESTOR_PUBLIC_KEY
 from api_testing_common import run_tests
 from api_testing_common import timestamp_to_isoformat
+from sci_testing_common import SCIBaseTest
 from protocol_constants import ProtocolConstants
 
+import requests
+
+from core.utils import calculate_maximum_download_time
+
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "concent_api.settings")
 
-(DIFFERENT_REQUESTOR_ETHEREUM_PRIVATE_KEY, DIFFERENT_REQUESTOR_ETHEREUM_PUBLIC_KEY) = generate_priv_and_pub_eth_account_key()
-(DIFFERENT_PROVIDER_ETHEREUM_PRIVATE_KEY, DIFFERENT_PROVIDER_ETHEREUM_PUBLIC_KEY) = generate_priv_and_pub_eth_account_key()
+
+REPORT_COMPUTED_TASK_SIZE = 10
```

Comment:
Why not imported from `api_common_testing`?

Reply:
Done